app/views/collections/index.html.erb

```diff
@@ -1,13 +1,13 @@
 <!--Descriptive page name and system messages, descriptions, and instructions.-->
 <h2 class="heading">
   <% if @user %>
-    <%= ts("%{user_name}'s Collections", :user_name => @user.login) %>
+    <%= t(".page_heading.user", user: @user.login) %>
```

Comment:
Based on
>  Furthermore, Translation finds it helpful for variable names in keys to closely correspond with the key for the text the variable represents.

from https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards#user-content-Locale_key_names, I wonder if it would be better to call these things like `users_collections` and `challenges_subcollections_in`?